glue/core/visual.py

```diff
@@ -94,6 +105,20 @@ def color(self, value):
         else:
             self._color = value
 
+    @callback_property
+    def preferred_cmap(self):
+        """
+        A preferred colormap specified using Matplotlib notation
+        """
+        return self._preferred_cmap
+
+    @preferred_cmap.setter
+    def preferred_cmap(self, value):
+        if isinstance(value, str):
+            self._preferred_cmap = self.convert_colormap_from_name(value)
```

Comment:
I think Matplotlib has a get_cmap method we could use here to avoid having to implement convert_colormap_from_name?

Reply:
Yup, I just tried out `get_cmap` and that worked as expected.